changelog/1.17.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "1.17.0"
+description: "Released on 03/17/2021"
+---
+
+### Breaking Changes ❗
+
+- helm: The `namespaceWhitelist` field on the Coder helm charts [has been
+  removed and the organization namespaces feature has been
+  deprecated](../admin/access-control/organizations.md#namespaces).
+  - Deployments that had this value set cannot add namespaces, and cannot remove
+    namespaces unless there are no environments still using them
+- Existing environments in whitelisted namespaces will continue to work
+- Environments cannot be created in whitelisted namespaces anymore
+
+### Features ✨
+
+- [Workspace Providers](../admin/workspace-providers/index.md): Workspace
+  providers enable a single Coder deployment to provision and manage workspaces
+  across multiple Kubernetes clusters and namespaces.
+  - web: New configuration page available via **Manage** > **Admin** >
+    **Workspace Providers**
+  - infra: New support for the use of multiple Kubernetes clusters and
+    namespaces
```

Comment:
Think we should also add a `web:` point for the new field in Create/Edit environment dialog